uwblueprint/paramedics-react
# Include CCP dashboard tab in url

[Notion ticket](https://www.notion.so/uwblueprintexecs/Changing-a-patient-profile-within-the-hospital-tab-brings-you-back-to-the-patient-overview-tab-c5df671329a44f369e836f274430739b)

## Changes
- Added CCP dashboard tab to url (i.e. ".../patientOverview" or ".../hospital") such that completing/cancelling the following actions lead back to the tab the user started the action from:
    - Add/Scan patient
    - Edit patient
    - Delete patient
    - Transport patient

## Testing
- Check the following flows to see that you are taken back to the correct tab when clicking "Cancel" or "Complete/Confirm" and the url contains "patientOverview" or "hospital" accordingly - NOTE: the default tab is "Patient Overview" so at least check for the "Hospital" tab
    - Scan patient
    - Edit patient
    - Delete patient
    - Transport patient (and click "View Patient Details" and make sure it keeps you on the same tab)
![image](https://user-images.githubusercontent.com/22457859/104551797-88825480-5605-11eb-9582-92a7a8fae354.png)

## Checklist
>[Code review doc for reference](https://www.notion.so/uwblueprintexecs/Code-Review-a21fc85b00394f488e92d9d605f6b2bc)

before opening PR

filling out PR

after opening PR
